docs/docusaurus/docs/reference/learn/data_quality_use_cases/distribution.md

```diff
@@ -1,8 +1,297 @@
+| 1004           | 504         | 80.00           | 2024-01-16        | Clothing         | 3.9            | 2024-02-01        |
+| 1005           | 505         | 3500.00         | 2024-01-17        | Electronics      | 4.6            | 2024-02-10        |
+
+In this dataset, `purchase_amount` represents the amount spent by customers in various `product_category`. Analyzing the distribution of `purchase_amount` and other numerical columns can reveal insights into customer behavior and detect anomalies.
+
+## Key distribution Expectations
+
+GX offers several Expectations specifically designed for managing data distribution. These can be added to an Expectation Suite via the GX Cloud UI or using the GX Core Python library.
+
+### Expect Column Values To Be Between
+
+Ensures that all values in a column fall between a specified minimum and maximum value.
+
+**Use Case**: Essential for bounding numerical values within valid ranges, such as ensuring product ratings or purchase amounts are within reasonable limits.
+
+```python
+gxe.expect_column_values_to_be_between(
+    column="product_rating",
+    min_value=1,
+    max_value=5,
+)
+```
+
+<small>View `ExpectColumnValuesToBeBetween` in the [Expectation Gallery](https://greatexpectations.io/expectations/expect_column_values_to_be_between).</small>
+
+### Expect Column Pair Values A To Be Greater Than B
+
+Validates that values in one column are greater than corresponding values in another column.
+
+**Use Case**: Crucial for maintaining valid relationships between data entities, like verifying that return dates are later than purchase dates.
+
+```python
+gxe.expect_column_pair_values_A_to_be_greater_than_B(
+    column_A="return_date",
+    column_B="purchase_date",
+)
+```
+
+<small>View `ExpectColumnPairValuesAToBeGreaterThanB` in the [Expectation Gallery](https://greatexpectations.io/expectations/expect_column_pair_values_A_to_be_greater_than_B).</small>
+
+### Expect Column Value Z Scores To Be Less Than
+
+Checks that the Z-scores (number of standard deviations from mean) of all values are below a threshold.
+
+**Use Case**: Powerful for identifying individual outliers and anomalous data points that could represent data entry issues or unusual transactions.
+
+```python
+gxe.expect_column_value_z_scores_to_be_less_than(
+    column="purchase_amount",
+    threshold=3,
+)
+```
+
+<small>View `ExpectColumnValueZScoresToBeLessThan` in the [Expectation Gallery](https://greatexpectations.io/expectations/expect_column_value_z_scores_to_be_less_than).</small>
+
+### Expect Column Quantile Values To Be Between
+
+Ensures that specified quantiles of a column fall within provided ranges.
+
+**Use Case**: Robustly monitors key statistics of the overall distribution, which is valuable for tracking metrics like median purchase amount or 90th percentile product ratings.
+
+```python
+gxe.expect_column_quantile_values_to_be_between(
```

Comment:
Just a note for when you move these snippets into a tested script - Expectations should be in 1.0 CamelCaseSyntax.